lib/sequenceserver/blast/formatter.rb

```diff
@@ -41,7 +41,7 @@ def run
           " -outfmt '#{format} #{specifiers}'" \
           " -out '#{file.path}' 2> /dev/null"
         logger.debug("Executing: #{command}")
-        system command
+        system(command, :chdir => '/home/alekhya/.sequenceserver')
```

Comment:
Ha. Your username is hardcoded here. Won't work for others. expand '~/.sequenceserver' and use that.